pkg/api/routes.go

```diff
@@ -96,6 +98,17 @@ func (rh *RouteHandler) SetupRoutes() {
 
 // Method handlers
 
+// CheckHealth godoc
+// @Summary Check Health API to check server health
+// @Description Check if zot server is up or not
+// @Router /v2/ [get]
+// @Accept json
+// @Accept json
+// @Success 200 {string} string "ok".
+func (rh *RouteHandler) CheckHealth(w http.ResponseWriter, r *http.Request) {
+	WriteData(w, http.StatusOK, "application/json", []byte{})
```

Comment:
what is the actual health check here? Just that http router and the port is working?

Reply:
Yes

Reply:
Do you want me to add more ?

Reply:
I think this is enough for starters, if we can't respond to this then an orchestrator should definitely restart us.